### PR TITLE
fix(Wallet/AssetList): Updated as per designs

### LINKED ISSUE #6481: Asset List needs to be updated to match designs
# Bug Report

## Description

- [x] on the left, first row, token name
- [x] on the left, second row, token balance + token symbol
- [x] on the right, balanced in selected fiat currency

**out of scope**: price change (should be blank or commented out)

current:

Screen Shot 2022-07-17 at 1 28 03 PM

expected:

Screen Shot 2022-07-17 at 1 28 14 PM

### PR DESCRIPTION
**Pull request by alexandraB99**
Closes #6481

### What does the PR do
Wallet/AssetList Updated as per designs
 
### Affected areas
Wallet/AssetList

### Screenshot of functionality (including design for comparison)
hh

FIGMA (price change is out of scope):
f

### CODE REVIEW

File: ui/imports/shared/controls/AssetDelegate.qml [hunk 1]
```diff
@@ -37,41 +37,35 @@ Item {
...
     StyledText {
         id: assetFullTokenName
-        text: name
         anchors.top: assetSymbol.bottom
         anchors.left: assetInfoImage.right
         anchors.leftMargin: Style.current.smallPadding
-        color: Style.current.secondaryText
         font.pixelSize: 15
+        color: Style.current.secondaryText
+        text: Utils.toLocaleString(enabledNetworkBalance, locale) + " " + symbol.toUpperCase()
```
Comment (caybro) [inline ui/imports/shared/controls/AssetDelegate.qml:58]: I think I've pointed out already elsewhere, so in short:
`qsTr("L%1 %2").arg(enabledNetworkBalance).arg(symbol)`

The full story: https://hackmd.io/@status-desktop/B1eOaf-nd/%2FjNzZnH8ZRiynrrXJ--XpmQ#Formatting-%E2%80%9Cstuff%E2%80%9D

Comment (caybro) [reply ui/imports/shared/controls/AssetDelegate.qml:58]: Or, you could simply write:
`enabledNetworkBalance.toLocaleCurrencyString(Qt.locale(), symbol)`

https://doc.qt.io/qt-5/qml-qtqml-number.html#toLocaleCurrencyString-method

File: ui/imports/shared/controls/AssetDelegate.qml [hunk 2]
```diff
@@ -37,41 +37,35 @@ Item {
...
-    }
-    StyledText {
-        id: assetCurrencyBalance
-        color: Style.current.secondaryText
-        text: Utils.toLocaleString(enabledNetworkCurrencyBalance.toFixed(2), locale) + " " + assetDelegate.currency.toUpperCase()
-        anchors.right: parent.right
-        anchors.rightMargin: 0
-        anchors.top: assetBalance.bottom
-        font.pixelSize: 15
+        text: assetDelegate.currencySymbol+Utils.toLocaleString(enabledNetworkCurrencyBalance.toFixed(2), locale)
```
Comment (caybro) [inline ui/imports/shared/controls/AssetDelegate.qml:69]: Same rant as above :)

File: ui/imports/shared/controls/AssetDelegate.qml (hunk 1 above)
Comment (caybro) [reply ui/imports/shared/controls/AssetDelegate.qml:58]: The whole point is that it's never up to the developer to decide where to put the symbol (before/after), or how to format the number ;)

**Review by caybro — CHANGES_REQUESTED** [review]
Some usual rants :) Sorry

**Review by alaibe — APPROVED** [review]

File: ui/imports/shared/controls/AssetDelegate.qml (hunk 1 above)
Comment (alexandraB99) [reply ui/imports/shared/controls/AssetDelegate.qml]: done, looks better now
tk

File: ui/imports/shared/controls/AssetDelegate.qml (hunk 2 above)
Comment (alexandraB99) [reply ui/imports/shared/controls/AssetDelegate.qml]: done, looks better now
tk

**Review by caybro — APPROVED** [review]
Looks good!

**Pull request merged**